packages/lib-classifier/src/components/Classifier/components/Layout/components/DefaultLayout/DefaultLayout.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import SubjectViewer from '../../../SubjectViewer'
+import ImageToolbar from '../../../ImageToolbar'
+import styled from 'styled-components'
+import { Box } from 'grommet'
+
+const boxStyles = {
+  pad: 'medium',
+  border: {
+    color: 'lightGrey',
+    side: 'all'
+  }
+}
+
+const ContainerGrid = styled.div`
```

Comment:
Grommet v2 has CSS grid layout components that may be of interest to use here: https://v2.grommet.io/grid

Reply:
Yeah, I tried these, but they aren't responsive without tracking the viewport size in the component. Which we could do?

Reply:
They have a responsive utility function, though, I haven't used it so I'm not sure if it'd be helpful here: https://v2.grommet.io/responsive

For CSS grid, we'll need to also think about an IE 11 fallback unfortunately. Grommet's grid has a `Grid.available` property we can check to know whether or not to render a fallback. The codesandbox for v2 being broken still makes this hard to test out.

Reply:
No, I'd really love to drop IE 11. Right now we're still defacto supporting it even though Edge is now two major versions beyond it because of the Google Analytics on browser use of the site.

Reply:
Okay. Until we have a documented decision on which browsers we're going to support, I suggest creating a separate issue to handle `Grid` fallback for IE11.

Reply:
Circling back to this sounds good depending on the outcome of the discussion on what to support.